FreydCategoriesForCAP/gap/LinearClosure.gi

```diff
@@ -150,6 +164,59 @@ InstallMethod( TwistedLinearClosure,
     
 end );
 
+##
+InstallMethod( LinearClosure,
```

Comment:
I suggest putting the `InstallMethod`s in the same order as the `DeclareOperation`s (or the other way round).

Reply:
The latest force-push seems to move some installations around, but the order of the declarations and the order of the installations still do not match.

Reply:
I thought you meant I should just put the installations using `IsCategoryOfRows` before the installations of `IsHomalgRing`. Otherwise, I retained the exact order of the existing code.